src/modules/core/containers/burger-menu/burger-menu.tsx

```diff
@@ -23,12 +23,30 @@ import type {
 import styles from './burger-menu.module.scss';
 
 const proActions: BurgerMenuAction[] = [
+  {
+    id: 'my-profile',
+    icon: 'user',
+    text: 'My profile',
+    variant: 'default',
+  },
+  {
+    id: 'my-bookings',
+    icon: 'list',
+    text: 'My bookings',
```

Comment:
Please add translations for this buttons (reuse the same translations from header component)